src/app/components/periscope/periscope.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, OnInit, ElementRef, Renderer2, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-periscope',
+  templateUrl: './periscope.component.html',
+  styleUrls: ['./periscope.component.scss']
+})
+export class PeriscopeComponent implements OnInit {
+    @Input() promoTitle: string;
+    @Input() buttonText: string;
+    @Input() imageSrc: string;
+    periscopeState: boolean;
+    backgroundImage: string;
+
+    constructor() { }
+
+    ngOnInit() {
+        this.backgroundImage = `url(${this.imageSrc})`;
+    }
+
+    movePeriscopeBackgroundUp (
```

Comment:
please change the name of the method to something more general, since it is doing more than moving the background-image up.